src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -1087,6 +1087,24 @@ public void setMentions(List<User> users, List<Member> members)
         return collection;
     }
 
+    private IPermissionContainer getPermissionContainer()
+    {
+        IPermissionContainer permChan;
+
+        // Check if the channel is a thread
+        if (isFromType(ChannelType.GUILD_NEWS_THREAD) || isFromType(ChannelType.GUILD_PUBLIC_THREAD) || isFromType(ChannelType.GUILD_PRIVATE_THREAD))
```

Comment:
You should use `ChannelType#isThread` for these checks usually, but using guild channel makes this helper method unnecessary